lib/cms/hippo_importer/base.rb

```diff
@@ -0,0 +1,99 @@
+module Cms
+  module HippoImporter
+    class Base
+      def initialize(data:, docs:, to: 'en', parser: HippoXmlParser, html_decoder: HTMLEntities.new)
+        @records = parser.parse(data, [hippo_type])
+        @_site = to || 'en'
```

Comment:
What's with the underscore prefix on these variables? Is it to avoid naming clashes?